.github/workflows/cmake.yml

```diff
@@ -23,6 +28,11 @@ jobs:
     env:
       CC: ${{ matrix.cc }}
       CXX: ${{ matrix.cxx }}
+      OMP_NUM_THREADS: 2
+      CONAN_SKIP_BROKEN_SYMLINKS_CHECK: 'True'
```

Comment:
Would it be better to just repair the broken link?

Reply:
If I remember correctly the link breaks because conan does not preserve the directory structure in the package, or something like that. Since it's just linking a README, I think it's ok to skip it.